wdio/specs/homepage.js

```diff
@@ -24,4 +24,97 @@ describe('Municipality homepage', () => {
     const showAllLink = browser.getText('div#elm-events-block a.btn-show-all');
     assert.equal('הצג הכל', showAllLink);
   });
+
+  it('should show user types labels in the selected language: hebrew', () => {
```

Comment:
@amitaibu I think I will keep it, this is one test to check that both labels are in hebrew.